pkg/tfpfbridge/schemashim/attr.go

```diff
@@ -15,10 +15,29 @@
 package schemashim
 
 import (
+	"fmt"
+	"reflect"
+
 	pfattr "github.com/hashicorp/terraform-plugin-framework/attr"
 )
 
 type attr interface {
+	attrLike
```

Comment:
Would it not be possible to combine `attr` and `attrLike`? Perhaps I'm missing some dependency implications?